src/open_inwoner/search/views.py

```diff
@@ -35,7 +35,14 @@ class SearchView(
     success_url = reverse_lazy("search:search")
 
     def page_title(self):
-        return _("Zoeken")
+        form = self.get_form()
+        if form.is_valid():
+            data = form.cleaned_data.copy()
+            query = data.pop("query")
+            if query:
+                return f"{query} {_('Zoeken')}"
```

Comment:
I think it would look nicer with a little hyphen between the query and Zoeken?
Everything else looks good.